### PR TITLE
Full description reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Not perfect, but good enough to be translated.

### CODE REVIEW

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -1,29 +1,29 @@
-Chinese checkers is a strategy board game that is the simplified variation of the popular Halma game, and your objective is to do whatever it takes to be the first player who races his pieces across the board. 
-Halma is an open-source Halma game that offers both the traditional 16x16 and the Chinese Checkers variant. The multiplayer feature enables you to play the checkers with up to 6 players. If you don’t want to play the multiplayer mode, you can challenge your checkers-solving skills against the AI.
-So, if you are into such strategy board games and looking for a perfect Halma experience, you’ve come to the right place. Download Halma for free on your Android device, choose your preferred board types and have fun playing Chinese checkers with family, friends, or other players all around the world.
+A simplified variation of the popular Halma game.
```
Comment (comradekingu) [inline fastlane/metadata/android/en-US/full_description.txt:1]:
```suggestion
```
This is the part I am the most unsure about.
If I don't know what Halma is, then what does a simplified version look like?
Not sure if Crazy Marvin Halma is different from the original or a custom variant…?

Comment (CrazyMarvin) [reply fastlane/metadata/android/en-US/full_description.txt:1]: Halma is is the traditional variation and Chinese Checkers (or Sternhalma) is the simplified version of it.
Crazy Marvin Halma offers both modes.

**Comment by CrazyMarvin** [comment]
Thank you very much for the rework! 😘

**Comment by CrazyMarvin** [comment]
I did some fine-tuning with 897b78f173aef059eb7b752693dcd0adc6ebb21f and think the description is now ready to be published. 😂

**Pull request merged**